Configure

```diff
@@ -1360,9 +1359,7 @@ unless ($disabled{"crypto-mdebug-backtrace"})
 	}
 
 if ($user_cflags ne "") { $config{cflags}="$config{cflags}$user_cflags"; $config{cxxflags}="$config{cxxflags}$user_cflags";}
-else                    { $no_user_cflags=1;  }
 if (@user_defines) { $config{defines}=[ @{$config{defines}}, @user_defines ]; }
```

Comment:
What's the point of these two if's now, without else's? I mean if $iser_cflags is empty, appending it would yield same string. I'd also argue in favour of `$config{cflags} .= $user_cflags`...

Reply:
Good point.  Fixing.